src/ert/shared/services/_base_service.py

```diff
@@ -103,6 +104,7 @@ def __init__(
         self._exec_args = exec_args
         self._timeout = timeout
         self._set_conn_info = set_conn_info
+        self._service_config_path = Path(project) / f"{self._service_name}_server.json"
```

Comment:
is it strictly speaking necessary to call `Path` on `project`? according to the function definition of `__init__`, `project` is already of type `Path`

Reply:
This is python and types in the function definitions are just hints, so it falls on the developer to make sure that what you are passing to the function is of a specified type, otherwise, bad things happen :). 

I can remove the explicit conversion to Path. Still, I need to make sure all the initialization of the BaseService is with an actual Path object and not a string, and what I get in the end is Path conversion in multiple other places, so I will change the expected type to string and only have the Path conversion here.